src/main/java/de/tum/in/www1/artemis/service/programming/ProgrammingExerciseImportService.java

```diff
@@ -95,18 +95,18 @@ public void importRepositories(final ProgrammingExercise templateExercise, final
 
         String sourceBranch = versionControl.getOrRetrieveBranchOfExercise(templateExercise);
 
+        // TODO: in case one of those operations fail, we should do error handling and revert all previous operations
         versionControl.copyRepository(sourceProjectKey, templateRepoName, sourceBranch, targetProjectKey, RepositoryType.TEMPLATE.getName());
         versionControl.copyRepository(sourceProjectKey, solutionRepoName, sourceBranch, targetProjectKey, RepositoryType.SOLUTION.getName());
         versionControl.copyRepository(sourceProjectKey, testRepoName, sourceBranch, targetProjectKey, RepositoryType.TESTS.getName());
 
-        List<AuxiliaryRepository> auxiliaryRepositories = templateExercise.getAuxiliaryRepositories();
-        for (int i = 0; i < auxiliaryRepositories.size(); i++) {
-            AuxiliaryRepository auxiliaryRepository = auxiliaryRepositories.get(i);
-            String repositoryUrl = versionControl
-                    .copyRepository(sourceProjectKey, auxiliaryRepository.getRepositoryName(), sourceBranch, targetProjectKey, auxiliaryRepository.getName()).toString();
-            AuxiliaryRepository newAuxiliaryRepository = newExercise.getAuxiliaryRepositories().get(i);
-            newAuxiliaryRepository.setRepositoryUrl(repositoryUrl);
-            auxiliaryRepositoryRepository.save(newAuxiliaryRepository);
+        List<AuxiliaryRepository> auxRepos = templateExercise.getAuxiliaryRepositories();
+        for (int i = 0; i < auxRepos.size(); i++) {
+            AuxiliaryRepository auxRepo = auxRepos.get(i);
+            var repoUrl = versionControl.copyRepository(sourceProjectKey, auxRepo.getRepositoryName(), sourceBranch, targetProjectKey, auxRepo.getName()).toString();
```

Comment:
```suggestion
            String repoUrl = versionControl.copyRepository(sourceProjectKey, auxRepo.getRepositoryName(), sourceBranch, targetProjectKey, auxRepo.getName()).toString();
```
Oder gibt es einen Grund `String` zu `var` zu verkürzen?

Reply:
My goal was that the code fits into one line in IntelliJ to avoid line breaks which - at least for me - make the code harder to read ;-)